server/backend/api/app/models/model.py

```diff
@@ -240,13 +376,13 @@ class FamUser(Base):
     )
 
     fam_user_role_xref = relationship("FamUserRoleXref", back_populates="user")
-    user_type_relation = relationship(
+    user_type_relation: Mapped[FamUserType] = relationship(
```

Comment:
just curious, do we need to change the user_type_relation?

Reply:
We don't need to change this one. I think for some reason before I touched it and forgot to change it back (but does not hurt to have it). I revert it back.